plugin/build.gradle

```diff
@@ -36,6 +36,18 @@ android {
 
     flavorDimensions = ["vendor"]
     productFlavors {
+        android {
+            dimension "vendor"
+            ndk {
+                //noinspection ChromeOsAbiSupport
+                abiFilters 'arm64-v8a', 'x86_64'
+            }
+            externalNativeBuild {
+                cmake {
+                    arguments "-DFLAVOR=android"
+                }
+            }
+        }
```

Comment:
Do we really need another AAR built? We've never used the `FLAVOR` define in the C++ anywhere - could we just use the Khronos AAR for Android too?

(The same goes for MagicLeap - I think we could reuse the Khronos AAR for that as well)

Reply:
Ah, I guess it's because the Java code is different.

Well, after PR https://github.com/GodotVR/godot_openxr_vendors/pull/227 is merged, this won't be as big of a deal because with that one we do only one C++ build, and then each of the Java builds just reuses the artifacts of that :-)

Reply:
Yes the new flavor is primarily for the difference in the java code and the manifest. The C++ code and configuration is the same as the others.